Lib/gftools/builder/operations/__init__.py

```diff
@@ -149,18 +151,42 @@ def variables(self):
 
         return vars
 
-
-known_operations = {}
-
-for mod in pkgutil.iter_modules([dirname(__file__)]):
-    imp = importlib.import_module("gftools.builder.operations." + mod.name)
-    classes = [
-        (name, cls)
-        for name, cls in inspect.getmembers(sys.modules[imp.__name__], inspect.isclass)
-        if "OperationBase" not in name and issubclass(cls, OperationBase)
-    ]
-    if len(classes) > 1:
-        raise ValueError(
-            f"Too many classes in module gftools.builder.operations.{mod.name}"
-        )
-    known_operations[mod.name] = classes[0][1]
+# just pretend these live elsewhere, here now for illustration
+class FontcBuildVariable(OperationBase):
+    description = "Build a variable font from a source file (with fontc)"
+    rule = "fontc -o $out $in"
+
+class FontcBuildTTF(OperationBase):
+    description = "Build a TTF from a source file (with fontc)"
+    rule = "fontc -o $out $in"
+
+class OperationRegistry:
+    def __init__(self, use_fontc: bool):
+        self.known_operations = get_known_operations()
+        if use_fontc:
+            self.known_operations["buildVariable"] = FontcBuildVariable
+            self.known_operations["buildTTF"] = FontcBuildTTF
+        print(self.known_operations, file=sys.stderr)
+
+    def get(self, operation_name: str):
+        return self.known_operations.get(operation_name)
+
+
+def get_known_operations() -> Dict[str, OperationBase]:
```

Comment:
Operation registry is basically a factory. If we had a different factory for fontc vs dynamicly loaded python, created based on flags, then only the python one would need to do the dynamic module import bit. 

Considering most things use the googlefonts implementation and it always makes the same choices I find myself wondering if there should be a static implementation of that but I suppose that's out of scope.

Reply:
I presume the dynamic loading stuff exists for a reason, and I'm not trying to make any big changes here; just override some of the operations in the presence of a flag.

The rest of the operation types we're happy to share.